[23/Apr/2014 15:16:08] ERROR [django.request:226] Internal Server Error: /home/
Traceback (most recent call last):
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/core/handlers/base.py", line 114, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)
  File "/Users/wendellpbarreto/Projects/python/fotec/fotec/apps/core/views.py", line 115, in get
    return self._request(request, args, kwargs)
  File "/Users/wendellpbarreto/Projects/python/fotec/fotec/apps/core/views.py", line 107, in _request
    return render_to_response(self.get_template_name(request), template_data, context_instance=RequestContext(request))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/shortcuts/__init__.py", line 29, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader.py", line 169, in render_to_string
    return t.render(context_instance)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 140, in render
    return self._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 134, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 123, in render
    return compiled_parent._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 134, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 305, in render
    return nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 196, in render
    nodelist.append(node.render(context))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 305, in render
    return nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 88, in render
    output = self.filter_expression.resolve(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 585, in resolve
    obj = self.var.resolve(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 735, in resolve
    value = self._resolve_lookup(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 789, in _resolve_lookup
    current = current()
  File "/Users/wendellpbarreto/Projects/python/fotec/fotec/apps/core/models.py", line 550, in video
    result = u"<iframe width='100%'' src=\"http://www.youtube.com/embed/%s\"></iframe>" % self.youtube
ValueError: unsupported format character '"' (0x22) at index 20
[23/Apr/2014 15:16:10] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:16:10] ERROR [django.request:226] Internal Server Error: /home/
Traceback (most recent call last):
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/core/handlers/base.py", line 114, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)
  File "/Users/wendellpbarreto/Projects/python/fotec/fotec/apps/core/views.py", line 115, in get
    return self._request(request, args, kwargs)
  File "/Users/wendellpbarreto/Projects/python/fotec/fotec/apps/core/views.py", line 107, in _request
    return render_to_response(self.get_template_name(request), template_data, context_instance=RequestContext(request))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/shortcuts/__init__.py", line 29, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader.py", line 169, in render_to_string
    return t.render(context_instance)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 140, in render
    return self._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 134, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 123, in render
    return compiled_parent._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 134, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 305, in render
    return nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 196, in render
    nodelist.append(node.render(context))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 305, in render
    return nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 88, in render
    output = self.filter_expression.resolve(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 585, in resolve
    obj = self.var.resolve(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 735, in resolve
    value = self._resolve_lookup(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 789, in _resolve_lookup
    current = current()
  File "/Users/wendellpbarreto/Projects/python/fotec/fotec/apps/core/models.py", line 550, in video
    result = u"<iframe width='100%'' src=\"http://www.youtube.com/embed/%s\"></iframe>" % self.youtube
ValueError: unsupported format character ''' (0x27) at index 19
[23/Apr/2014 15:16:31] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:17:06] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:17:06] ERROR [django.request:226] Internal Server Error: /home/
Traceback (most recent call last):
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/core/handlers/base.py", line 114, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)
  File "/Users/wendellpbarreto/Projects/python/fotec/fotec/apps/core/views.py", line 115, in get
    return self._request(request, args, kwargs)
  File "/Users/wendellpbarreto/Projects/python/fotec/fotec/apps/core/views.py", line 107, in _request
    return render_to_response(self.get_template_name(request), template_data, context_instance=RequestContext(request))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/shortcuts/__init__.py", line 29, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader.py", line 169, in render_to_string
    return t.render(context_instance)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 140, in render
    return self._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 134, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 123, in render
    return compiled_parent._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 134, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 305, in render
    return nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 196, in render
    nodelist.append(node.render(context))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 305, in render
    return nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 88, in render
    output = self.filter_expression.resolve(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 585, in resolve
    obj = self.var.resolve(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 735, in resolve
    value = self._resolve_lookup(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 789, in _resolve_lookup
    current = current()
  File "/Users/wendellpbarreto/Projects/python/fotec/fotec/apps/core/models.py", line 550, in video
    result = u"<iframe width=\"\%\" src=\"http://www.youtube.com/embed/%s\"></iframe>" % ("100%", self.youtube)
ValueError: unsupported format character '"' (0x22) at index 17
[23/Apr/2014 15:17:11] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:17:11] ERROR [django.request:226] Internal Server Error: /home/
Traceback (most recent call last):
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/core/handlers/base.py", line 114, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)
  File "/Users/wendellpbarreto/Projects/python/fotec/fotec/apps/core/views.py", line 115, in get
    return self._request(request, args, kwargs)
  File "/Users/wendellpbarreto/Projects/python/fotec/fotec/apps/core/views.py", line 107, in _request
    return render_to_response(self.get_template_name(request), template_data, context_instance=RequestContext(request))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/shortcuts/__init__.py", line 29, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader.py", line 169, in render_to_string
    return t.render(context_instance)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 140, in render
    return self._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 134, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 123, in render
    return compiled_parent._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 134, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 305, in render
    return nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 196, in render
    nodelist.append(node.render(context))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 305, in render
    return nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 88, in render
    output = self.filter_expression.resolve(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 585, in resolve
    obj = self.var.resolve(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 735, in resolve
    value = self._resolve_lookup(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 789, in _resolve_lookup
    current = current()
  File "/Users/wendellpbarreto/Projects/python/fotec/fotec/apps/core/models.py", line 550, in video
    result = u"<iframe width=\"%\" src=\"http://www.youtube.com/embed/%s\"></iframe>" % ("100%", self.youtube)
ValueError: unsupported format character '"' (0x22) at index 17
[23/Apr/2014 15:17:16] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:17:16] ERROR [django.request:226] Internal Server Error: /home/
Traceback (most recent call last):
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/core/handlers/base.py", line 114, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)
  File "/Users/wendellpbarreto/Projects/python/fotec/fotec/apps/core/views.py", line 115, in get
    return self._request(request, args, kwargs)
  File "/Users/wendellpbarreto/Projects/python/fotec/fotec/apps/core/views.py", line 107, in _request
    return render_to_response(self.get_template_name(request), template_data, context_instance=RequestContext(request))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/shortcuts/__init__.py", line 29, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader.py", line 169, in render_to_string
    return t.render(context_instance)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 140, in render
    return self._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 134, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 123, in render
    return compiled_parent._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 134, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 305, in render
    return nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 196, in render
    nodelist.append(node.render(context))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 305, in render
    return nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 88, in render
    output = self.filter_expression.resolve(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 585, in resolve
    obj = self.var.resolve(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 735, in resolve
    value = self._resolve_lookup(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 789, in _resolve_lookup
    current = current()
  File "/Users/wendellpbarreto/Projects/python/fotec/fotec/apps/core/models.py", line 550, in video
    result = u"<iframe width=\"\\%\" src=\"http://www.youtube.com/embed/%s\"></iframe>" % ("100%", self.youtube)
ValueError: unsupported format character '"' (0x22) at index 16
[23/Apr/2014 15:17:22] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:17:22] ERROR [django.request:226] Internal Server Error: /home/
Traceback (most recent call last):
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/core/handlers/base.py", line 114, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)
  File "/Users/wendellpbarreto/Projects/python/fotec/fotec/apps/core/views.py", line 115, in get
    return self._request(request, args, kwargs)
  File "/Users/wendellpbarreto/Projects/python/fotec/fotec/apps/core/views.py", line 107, in _request
    return render_to_response(self.get_template_name(request), template_data, context_instance=RequestContext(request))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/shortcuts/__init__.py", line 29, in render_to_response
    return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader.py", line 169, in render_to_string
    return t.render(context_instance)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 140, in render
    return self._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 134, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 123, in render
    return compiled_parent._render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 134, in _render
    return self.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 305, in render
    return nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 196, in render
    nodelist.append(node.render(context))
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/defaulttags.py", line 305, in render
    return nodelist.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/debug.py", line 88, in render
    output = self.filter_expression.resolve(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 585, in resolve
    obj = self.var.resolve(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 735, in resolve
    value = self._resolve_lookup(context)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/template/base.py", line 789, in _resolve_lookup
    current = current()
  File "/Users/wendellpbarreto/Projects/python/fotec/fotec/apps/core/models.py", line 550, in video
    result = u"<iframe width=\"'%'\" src=\"http://www.youtube.com/embed/%s\"></iframe>" % ("100%", self.youtube)
ValueError: unsupported format character '"' (0x22) at index 17
[23/Apr/2014 15:17:55] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:23:37] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:24:55] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:25:51] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:27:37] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:28:31] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:28:57] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:29:52] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:30:39] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:31:02] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:31:10] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:32:22] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:32:59] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:33:32] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:34:02] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:34:59] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:36:14] WARNING [django.request:145] Not Found: /favicon.ico
[23/Apr/2014 15:36:30] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:36:58] WARNING [django.request:145] Not Found: /favicon.ico
[23/Apr/2014 15:37:11] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:38:41] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:39:16] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:40:06] WARNING [django.request:145] Not Found: /favicon.ico
[23/Apr/2014 15:40:22] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:43:16] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:43:23] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:43:31] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:43:37] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:43:43] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:44:07] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:44:12] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 15:44:18] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:01:46] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:02:08] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:04:15] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:06:12] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:06:23] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:06:56] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:09:37] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:10:14] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:14:38] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:20:03] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:22:07] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:22:50] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:25:28] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:25:40] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:25:45] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:27:55] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:27:58] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:29:50] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:31:18] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:33:09] INFO [fotec.apps.core.views:171] Template loaded: gui/notice.html
[23/Apr/2014 17:33:16] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:34:03] ERROR [fotec.apps.core.views:169] Template not found! Raised: gui/videolibrary.html
[23/Apr/2014 17:34:03] ERROR [fotec.apps.core.views:169] Template not found! Raised: gui.html
[23/Apr/2014 17:34:03] ERROR [fotec.apps.core.views:169] Template not found! Raised: gui/videolibrary.html
[23/Apr/2014 17:34:03] INFO [fotec.apps.core.views:175] Not found available templates, loading 404 template!
[23/Apr/2014 17:34:03] WARNING [django.request:145] Not Found: /favicon.ico
[23/Apr/2014 17:34:09] ERROR [fotec.apps.core.views:169] Template not found! Raised: gui/notice.html
[23/Apr/2014 17:34:09] ERROR [fotec.apps.core.views:169] Template not found! Raised: gui.html
[23/Apr/2014 17:34:09] ERROR [fotec.apps.core.views:169] Template not found! Raised: gui/notice.html
[23/Apr/2014 17:34:09] INFO [fotec.apps.core.views:175] Not found available templates, loading 404 template!
[23/Apr/2014 17:34:14] INFO [fotec.apps.core.views:171] Template loaded: gui/notice_read.html
[23/Apr/2014 17:36:03] INFO [fotec.apps.core.views:171] Template loaded: gui/notice_read.html
[23/Apr/2014 17:36:23] INFO [fotec.apps.core.views:171] Template loaded: gui/notice_read.html
[23/Apr/2014 17:36:29] INFO [fotec.apps.core.views:171] Template loaded: gui/notice_read.html
[23/Apr/2014 17:36:37] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:37:34] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:37:37] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:38:26] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:38:31] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:40:38] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:40:44] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:40:55] WARNING [django.request:145] Not Found: /favicon.ico
[23/Apr/2014 17:41:12] INFO [fotec.apps.core.models:471] Photo matching query does not exist.
[23/Apr/2014 17:41:12] ERROR [fotec.apps.core.models:423] [Errno 2] No such file or directory: '/Users/wendellpbarreto/Projects/python/fotec/media/photogallery/1'
[23/Apr/2014 17:41:14] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:42:12] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:42:47] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:46:27] WARNING [django.request:145] Not Found: /favicon.ico
[23/Apr/2014 17:53:59] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:55:49] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:55:52] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:56:32] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:57:49] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 17:58:19] WARNING [django.request:145] Not Found: /favicon.ico
[23/Apr/2014 17:59:16] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:00:10] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:00:22] WARNING [django.request:145] Not Found: /favicon.ico
[23/Apr/2014 18:05:28] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:05:30] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:05:32] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:05:42] WARNING [django.request:145] Not Found: /favicon.ico
[23/Apr/2014 18:06:44] ERROR [django.request:226] Internal Server Error: /admin/core/event/add/
Traceback (most recent call last):
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/core/handlers/base.py", line 114, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/contrib/admin/options.py", line 432, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/utils/decorators.py", line 99, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/decorators/cache.py", line 52, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/contrib/admin/sites.py", line 198, in inner
    return view(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/utils/decorators.py", line 29, in _wrapper
    return bound_func(*args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/utils/decorators.py", line 99, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/utils/decorators.py", line 25, in bound_func
    return func(self, *args2, **kwargs2)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/db/transaction.py", line 339, in inner
    return func(*args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/contrib/admin/options.py", line 1133, in add_view
    self.log_addition(request, new_object)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/contrib/admin/options.py", line 599, in log_addition
    object_repr=force_text(object),
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/utils/encoding.py", line 100, in force_text
    s = s.__unicode__()
  File "/Users/wendellpbarreto/Projects/python/fotec/fotec/apps/core/models.py", line 599, in __unicode__
    return "%s" % (self.name.capitalize())
AttributeError: 'Event' object has no attribute 'name'
[23/Apr/2014 18:06:56] ERROR [django.request:226] Internal Server Error: /admin/core/event/add/
Traceback (most recent call last):
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/core/handlers/base.py", line 114, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/contrib/admin/options.py", line 432, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/utils/decorators.py", line 99, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/views/decorators/cache.py", line 52, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/contrib/admin/sites.py", line 198, in inner
    return view(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/utils/decorators.py", line 29, in _wrapper
    return bound_func(*args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/utils/decorators.py", line 99, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/utils/decorators.py", line 25, in bound_func
    return func(self, *args2, **kwargs2)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/db/transaction.py", line 339, in inner
    return func(*args, **kwargs)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/contrib/admin/options.py", line 1133, in add_view
    self.log_addition(request, new_object)
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/contrib/admin/options.py", line 599, in log_addition
    object_repr=force_text(object),
  File "/Users/wendellpbarreto/.virtualenvs/fotec/lib/python2.7/site-packages/django/utils/encoding.py", line 100, in force_text
    s = s.__unicode__()
  File "/Users/wendellpbarreto/Projects/python/fotec/fotec/apps/core/models.py", line 599, in __unicode__
    return "%s" % (self.name.capitalize())
AttributeError: 'Event' object has no attribute 'name'
[23/Apr/2014 18:09:51] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:10:13] WARNING [django.request:145] Not Found: /favicon.ico
[23/Apr/2014 18:14:20] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:16:50] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:18:15] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:19:29] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:19:32] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:19:35] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:19:43] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:20:25] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:21:13] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:21:21] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:21:39] WARNING [django.request:145] Not Found: /favicon.ico
[23/Apr/2014 18:21:58] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:23:14] WARNING [django.request:145] Not Found: /favicon.ico
[23/Apr/2014 18:24:03] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:32:04] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:32:53] WARNING [django.request:145] Not Found: /favicon.ico
[23/Apr/2014 18:40:06] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:40:31] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:42:08] WARNING [django.request:145] Not Found: /favicon.ico
[23/Apr/2014 18:42:48] WARNING [django.request:145] Not Found: /favicon.ico
[23/Apr/2014 18:42:49] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:43:02] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:43:48] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:45:08] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:45:10] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:45:12] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:46:59] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:48:30] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:48:37] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:48:42] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:48:49] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:49:18] WARNING [django.request:145] Not Found: /favicon.ico
[23/Apr/2014 18:50:09] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:50:38] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:53:43] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:57:46] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 18:59:41] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 19:00:35] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 19:00:59] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 19:01:32] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 19:01:46] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 19:02:33] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 19:08:28] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 19:09:17] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 19:11:50] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
[23/Apr/2014 19:12:00] INFO [fotec.apps.core.views:171] Template loaded: gui/home.html
